lib/analyzer/binary-version-extractors/openjdk-jre.ts

```diff
@@ -30,15 +30,19 @@ function parseOpenJDKBinary(fullVersionOutput: string) {
   */
   const jdkVersionLines = fullVersionOutput &&
                           fullVersionOutput.trim().split('\n');
-  if (!jdkVersionLines || jdkVersionLines.length !== 3) {
+  if (!jdkVersionLines) {
     return null;
   }
   const bracketsRE = /\(build (.*)\)$/;
-  const buildVersion = jdkVersionLines[1].match(bracketsRE);
-  const version = buildVersion && buildVersion[1];
+  const runtimeEnv = 'Runtime Environment';
+  const buildVersion =
+    (jdkVersionLines.filter(line => line.includes(runtimeEnv)).pop() || '')
```

Comment:
I think you can use `.find` instead of `.filter` and remove the `pop`

Reply:
Maybe also moving this logic to line `32` and let the `if` discard lines without `Runtime environment`.

Reply:
Nice! I like this much better